source/docs/software/dashboards/labview-dashboard/index.rst

```diff
@@ -1,6 +1,8 @@
 LabVIEW Dashboard
 =================
 
+The LabVIEW Dashboard is easy to use and provides a lot of featues straight out of the box like: camera streams, autonomous selection, and joystick feedback.  It can be customized using LabVIEW by creating a new Dashboard project.  While it :ref:`can be used <docs/software/dashboards/labview-dashboard/using-the-labview-dashboard-with-c++-java-code:Using the LabVIEW Dashboard with C++/Java Code>` by Java or C++ teams they generally prefer SmartDashboard or Shuffleboard which can be customized in their respective language.
```

Comment:
```suggestion
The LabVIEW Dashboard is easy to use and provides a lot of featues straight out of the box like: camera streams, autonomous selection, and joystick feedback.  It can be customized using LabVIEW by creating a new Dashboard project.  While it :ref:`can be used <docs/software/dashboards/labview-dashboard/using-the-labview-dashboard-with-c++-java-code:Using the LabVIEW Dashboard with C++/Java Code>` by Java or C++ teams, they generally prefer SmartDashboard or Shuffleboard which can be customized in their respective language.
```